libmscore/textedit.cpp

```diff
@@ -455,6 +456,11 @@ bool TextBase::edit(EditData& ed)
                         }
                   }
             }
+            if (ctrlPressed && altPressed) {
+                  if (ed.key == Qt::Key_hyphen) {
+                        s = "\u203f"; // Unicode undertie
```

Comment:
I guess weÄd better use `"<sym>lyricsElision</sym>`" here

Reply:
Or even
```
      insertSym(ed, SymId::lyricsElision);
      return true;
```

Reply:
I will test when you want, I have many scores with Elision, and if success, at least 2 issues could be fixed.

Reply:
I just tried i.e. 'manually', i.e. hand-editing an mscx file, and it works

Reply:
But whether my code above works remains to be tested ;-)

Reply:
Really good☺

Reply:
With the last commit I changed the symbol from unicode to smufl. As I originally used unicode because flat, sharp and natural symbols were drawn from unicode, I wonder whether those symbols should be changed too.